### PR TITLE
900 sprint02 comments

### PR DESCRIPTION
**Pull request by Enlarem**
Added : 
- Dropdown to select the country for the phone number
- Autocomplete element for the country selection
- Field validations in function of the different country selected (for the phone and postal code)
- Labels on login and register pages
- Grid layout for the Register page
- Password double check before registering

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ require("dotenv").config();
 var cors = require("cors");
 const { insertSampleData, deleteAllDocuments } = require("./bin/defaultdata");
 
-const mongoDB = process.env.MONGODB_CONN_STRING;
+// const mongoDB = process.env.MONGODB_CONN_STRING;  
+const mongoDB = "mongodb://localhost:27017/assoceasedb";
```
Comment (Nesssu) [inline server/app.js:11]: This should probably be changed back to the "correct" way.

**Comment by jannejjj** [comment]
There are problems with how the postal code and phone number validation is currently done, since it doesn't support all countries in the world, unsurprisingly :D I'm investigating a fix

File: server/app.js (hunk 1 above)
Comment (Enlarem) [reply server/app.js:11]: True true, I'm modifying this

**Review by Nesssu — APPROVED** [review]

**Review by jannejjj — APPROVED** [review]
After the changes, looks good! Also tested registration, validation, and displaying the information in My Profile, seems to work nicely

There are some really small things that could be changed, such as the validation functions not needing to actually return anything, but eh let's not worry about such small details :D

**Review by Jezerig — APPROVED** [review]

**Pull request merged**